.travis.yml

```diff
@@ -2,8 +2,7 @@
 language: julia
 os:
   - linux
-  # We don't have docker on OSX on travis at the moment...
-  #- osx
+  - osx
```

Comment:
I suspect this won't work due to the lack of nested virtualization in the travis environment.

Reply:
Well it didn't work, but it seems to be because of a segfault from another package?  That's weird.

Reply:
It would also be really too bad if this didn't work on Travis, as that would eliminate a lot of the usefulness of this work, with regards to licensing.

Reply:
Yeah, I know, I didn't find out that Travis disabled this until I was already most of the way through :/